accounts/templates/account/mentor/home.html

```diff
@@ -50,7 +50,7 @@ <h2 class="title text-tertiary">Your Profile</h2>
                 {% if mentor.avatar and not mentor.avatar_approved %}
                     <small class="text-tertiary">Pending Approval</small>
                 {% endif %}
-                <img class="avatar thumbnail" src="{% if mentor.avatar.thumbnail %}{{ mentor.avatar.thumbnail.url }}{% else %}https://gravatar.com/avatar/?s=283&amp;d=mm{% endif %}" alt="">
+                <img class="avatar thumbnail" src="{{ mentor.get_avatar.thumbnail.url }}" alt="">
```

Comment:
The deleted gravatar url had a query string parameter that was passing a size of 283 (`s=283`. If `get_avatar` does end up returning a gravatar (when mentor has no photo) it will end up being sized at 320 based on how things are setup in [coderdojochi/models/mentor.py](https://github.com/WeAllCode/website/blob/412fa6cc3a9f12c89958b6f093956c648ddd69f0/coderdojochi/models/mentor.py#L138).

Should we modify get_avatar to accept a size parameter instead of the currently hardcoded [320](https://github.com/WeAllCode/website/blob/412fa6cc3a9f12c89958b6f093956c648ddd69f0/coderdojochi/models/mentor.py#L159) or [500](https://github.com/WeAllCode/website/blob/412fa6cc3a9f12c89958b6f093956c648ddd69f0/coderdojochi/models/mentor.py#L166)? It's possible this could be important, like in instances where width/height are set in the image tag, as in [coderdojochi/templates/volunteer.html](https://github.com/WeAllCode/website/blob/412fa6cc3a9f12c89958b6f093956c648ddd69f0/coderdojochi/templates/volunteer.html#L32) for example?